src/content/docs/alerts-applied-intelligence/new-relic-alerts/get-started/your-first-nrql-condition.mdx

```diff
@@ -10,127 +10,276 @@ redirects:
+
+  </Collapser>
+</CollapserGroup>
+
+## Connect your condition to a policy [#connect-condition-policy]
+
+  If there are any latency issues with web transaction times, we'd like to receive a notification as soon as possible. The best quick and efficient action is to create an alert condition that will open an incident if web transaction times take too long.
+ 
+  This alert condition is a container that holds all the rules—are we using static or anomaly thresholds, are we using a sliding-window aggregation or just leaving the evaluation period as normal? 
+
+  At this point in the process we now have a fully defined container and we've set all the rules to make sure an incident is opened when we want it to be. Based on the settings above, if our alert condition recognizes this behavior in our system that "violates" the thresholds that we've set, it will create an incident. Now, all we need to do is to attach this container to a policy. 
+  
+  The policy is the sorting system for the incident. When you create a policy, you're creating the tool that organizes all of your incoming incidents. You can connect policies to **[worflows](/docs/alerts-applied-intelligence/applied-intelligence/incident-workflows/incident-workflows/)** that tell New Relic where you want all this incoming information to go, how often you want it to be sent, and where. 
+
+<CollapserGroup>
+  <Collapser
+    id="select-an-existing-policy"
+    title="Select an existing policy"
+  >
+
+  If you already have a policy that you want to connect your alert condition too, then here's where you would select the existing policy. 
+
+  Learn more about how to create policies [here](/docs/alerts-applied-intelligence/new-relic-alerts/alert-policies/specify-when-alerts-create-incidents/).
+  </Collapser>
+
+  <Collapser
+    id="create-a-new-policy"
+    title="Create a new policy"
+  >
+    If we want to create a new policy for this alert condition, here's our chance. Remember, a policy is connected to workflows and workflows control how often we're notified about any incidents. It's a fine balance between ensuring that we learn about any issues with web transaction time as quickly as possible and making sure that we don't get so many alerts that our developers experience fatigue and start missing out on important information because of information overload. 
+
+    Policies can hold multiple conditions or just one. In our case, if we're looking to make sure we learn about web transaction latency we have a few options. 
+    
+    First, we could create a policy that only attaches a single issue per policy, our default option. One issue per policy reduces noise but also requires immediate action. But this means that if we've multiple conditions attached to this policy, not just this one, **Response time: Example app**, then no matter what, **all incidents in this policy will be grouped into one single issue**.
```

Comment:
```suggestion
    First, you could create a policy that only attaches a single issue per policy (the default option). One issue per policy reduces noise but also requires immediate action. But this means that if you've attached multiple conditions to this policy, not just to **Response time: Example app**, then no matter what, **all incidents in this policy will be grouped into one single issue**.
```

I think this needs to be reworded. It's using what seems like a UI element (**all incidents in this policy will be grouped into one single issue**) as a part of the sentence.